deploy/manifests/dev/us-east-2/tenant/storetheindex/indexstar/deployment.yaml

```diff
@@ -19,8 +19,8 @@ spec:
             - '--backends=http://ber-indexer:3000/'
             - '--backends=http://cali-indexer:3000/'
             - '--backends=http://ago-indexer:3000/'
```

Comment:
If ago is not running a valuestore (only writing to dhstore), then I think I should remove it from the indexstar config. Is that correct?

Reply:
Right. I think this should be the corresponding dhfind endpoint if not included already.